### PR TITLE
Refine abstract from review

### PR DESCRIPTION
**Pull request by minrk**
- make value to policymakers clearer
- expand HPC
- trade consortium for training, keeping to length limit

### CODE REVIEW

File: abstract.tex [hunk 1]
```diff
@@ -18,7 +18,7 @@
 that can be run easily by anyone.
 As institutions and policies increasingly adopt Open and Reproducible policies,
 researchers urgently need practical tools to fulfill these requirements.
-Further, policy makers can benefit from tools for evaluation and enforcement.
+Further, policy makers can benefit from tools for evaluation and enforcement of reproducibility policies, once established.
```
Comment (annefou) [inline abstract.tex]: Why did you add "once established"? 
Do you mean that outcomes of the SOURCE project could become "policy instruments" and could encourage institutions, research councils, etc. to adopt and enforce Open and Reproducible policies?

Comment (fangohr) [reply abstract.tex]: I think that 'once established' refers to the policies that the policymakers will establish: once they have this, Binder tools can be used to evaluate if people stick to them, or enforce them (by checking the evaluation).

But even if the above is the intended interpretation, I'd think we could drop ", once established".

**Comment by fangohr** [comment]
Open question `, once established`could be dropped - or we need to make the intention behind it clearer.

All other changes I think are good.

There is a part in the middle where it doesn't flow for me yet (I'll create another PR when I get to this). 

I also think we need to get rid of all formatting in the abstract before we copy and paste, to the online form, so it would be better to get rid of the emphasis now (because the panel will not see that emphasis later).

**Review by fangohr — APPROVED** [review]
